RecommendationsWebAPI/Controllers/UserController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DataAccess.Interfaces;
+using DataAccess.Models;
+using Microsoft.AspNetCore.Mvc;
+using RecommendationsWebAPI.DTOs;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace RecommendationsWebAPI.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class UserController : ControllerBase
+{
+    private readonly IUserDataAccess _userDataAccess;
+
+    public UserController(IUserDataAccess userDataAcess)
+    {
+        _userDataAccess = userDataAcess;
+    }
+
+    // GET api/<UserController>/5
+    [HttpGet("{id}")]
+    public async Task<IActionResult> GetUserById(int id)
+    {
+        var user = DTOConverter<User, UserDTO>.From(await _userDataAccess.GetByIdAsync(id));
+
+        if (user == null)
```

Comment:
I think var movie is of type `User?` so it should have `.HasValue()` method instead of using `== null`, if it is of type `User`, discard this comment